HedgeModManager/Languages/pt-PT.xaml

```diff
@@ -211,12 +211,12 @@
     <system:String x:Key="ExceptionWindowCrash"           >Hedge Mod Manager Lançou uma Exceção!</system:String>
     <system:String x:Key="ExceptionWindowUnhandled"       >Hedge Mod Manager Encontrou um Erro!</system:String>
     <system:String x:Key="ExceptionWindowButtonCopyLog"   >Copiar Log para Área de Transferência</system:String>
-    <system:String x:Key="ExceptionWindowButtonIgnore"    >Ignorar and Fechar</system:String>
+    <system:String x:Key="ExceptionWindowButtonIgnore"    >Ignorar e Fechar</system:String>
     <system:String x:Key="ExceptionWindowButtonReport"    >Submeter Relatório no GitHub</system:String>
     <system:String x:Key="ExceptionWindowNewUpdateTitle"  >Nova Atualização Disponível!</system:String>
-    <system:String x:Key="ExceptionWindowNewUpdateText" xml:space="preserve" >Parece que esta versão do Hedge Mod Manager encontra-se desatualizada.&#x0a;É altamente recomendado que atualizes primeiro e verifiques se este problema ainda ocorre.&#x0a;Queres atualizar agora?</system:String>
```

Comment:
For some reason this doesn't sound right to me. "se encontra" should indeed be used when referring to something (ex. "Parece que se encontra desatualizada"), but with the mod manager's name in the middle, "encontra-se" just comes out better to me. Care to explain this change?

Reply:
Well, "se econtra" just sounded more natural to me (maybe an Azorean thing).
However since you mentioned this I went to take a further look at it.
While it can be used both ways, your "encontra-se" is the most grammatically accurate since there's no adverb preceeding it.